src/tasks/lint-locales.js

```diff
@@ -0,0 +1,32 @@
+const gulp = require('gulp');
+const {runAll} = require('@shopify/theme-lint');
+
+const config = require('./includes/config.js');
+const Reporter = require('./includes/lint-reporter.js').default;
+
+/**
+ * Runs all the translation tests and the reporter outputs
+ * the locale results once completed.
+ *
+ * @returns {String} Finalized linting output
+ * @private
+ */
+function lintLocales() {
+  return runAll(config.src.root, new Reporter()).then((reporter) => {
+    return reporter.finalize();
+  }).catch((err) => {
```

Comment:
It would if you call `Promise.reject()` inside of the `runAll` function. Will leave this out for now.